tests/components/myio/test_config_flow.py

```diff
@@ -0,0 +1,132 @@
+"""Test the myIO config flow."""
+from asynctest import patch
+
+from homeassistant import config_entries, setup
+from homeassistant.components.myio.config_flow import (
+    AppPortProblem,
+    CannotConnect,
+    InvalidAuth,
+)
+from homeassistant.components.myio.const import DOMAIN
+
+
+async def test_form(hass):
+    """Test we get the form."""
+    await setup.async_setup_component(hass, "persistent_notification", {})
+    result = await hass.config_entries.flow.async_init(
+        DOMAIN, context={"source": config_entries.SOURCE_USER}
+    )
+    assert result["type"] == "form"
```

Comment:
We have a constant for this in `homeassistant.data_entry_flow`:

```suggestion
    assert result["type"] == data_entry_flow.RESULT_TYPE_FORM
```

Reply:
changed.